MapboxDirections/MBRouteOptions.swift

```diff
@@ -111,6 +118,35 @@ open class RouteOptions: DirectionsOptions {
      Currently, you can only specify a single road class to avoid.
      */
     @objc open var roadClassesToAvoid: RoadClasses = []
+    
+    /**
+     A bias which determines whether the route should prefer or avoid the use of alleys. The
+     allowed range of values is from -1.0 to 1.0, where -1 indicates preference to avoid
+     alleys, 1 indicates preference to favor alleys, and 0 indicates no preference.
+     
+     Defaults to 0.
+     This property has no effect unless profile identifier is set to `MBDirectionsProfileIdentifier.walking`.
+     */
+    @objc open var alleyBias: Double = 0
+    
+    /**
+     A bias which determines whether the route should prefer or avoid the use of roads or paths
+     that are set aside for pedestrian-only use (walkways). The allowed range of values is from
+     -1.0 to 1.0, where -1 indicates indicates preference to avoid walkways, 1 indicates preference
+     to favor walkways, and 0 indicates no preference.
+     
+     Defeaults to 0.
+     This property has no effect unless profile identifier is set to `MBDirectionsProfileIdentifier.walking`.
+     */
+    @objc open var walkwayBias: Double = 0
+    
+    /**
+     Walking speed in meters per second. Must be between 0.14 and 6.94 meters per second.
```

Comment:
Let’s add constants or enumeration cases for these preset values (same for the other properties). Otherwise, there’s a risk that magic numbers could float around in obscure client-side code, only to break when things change on the server side.